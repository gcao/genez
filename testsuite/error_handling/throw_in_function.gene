# Test throw inside function
(fn may_fail []
  (println "Function called")
  (throw "Function error"))

(try
  (may_fail)
  (catch e
    (println "Caught from function:" e)))