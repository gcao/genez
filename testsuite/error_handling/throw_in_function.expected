Function called
Caught from function: Function error